pkg/controller/clusterrole/clusterrole_controller.go

```diff
@@ -1,10 +1,12 @@
+// Package clusterrole implements a controller for watching ClusterRole resources and tracking them.
+// Deprecated: Since we don't need to track ClusterRole objects anymore this is deprecated.
```

Comment:
I'm going to keep this as it because I just want the controller to cleanup the finalizers and the exporter is just deployed in staging for now. So performance is not an issue.